cfgFunctions.hpp

```diff
@@ -30,7 +32,8 @@ class GRAD_replay {
 	class server {
 		file = MODULES_DIRECTORY\@gruppe-adler\replay\functions\server;
 
-		class getSideColor {};
+        class canTrackUnit {};
+		class getSideColorID {};
```

Comment:
what's up with the indentation here and in line 33?

Reply:
Tabs mixed with spaces... which is weird, because both `canTrackUnit` and `getSideColorID` are part of this PR and it's not like i switched editors in between.

Reply:
Ah, `tab type` was set to `auto`.

Reply:
Matter of fact @nomisum , looks like you are using tabs while I'm using spaces. I'm gonna convert it all to tabs.

Reply:
But spaces > tabs :U